.github/workflows/publish-dbt-templater-release-to-pypi.yaml

```diff
@@ -4,6 +4,7 @@ on:
   release:
     types:
       - published
+  workflow_dispatch:
```

Comment:
Following our discussion in the Docker PR I figure it might be worth adding these to the PyPI publish workflows as well so we can re-run manually in these situations? 
Alternatively can remove and do a patch release if you prefer. Although, no package code has changed since 0.8.2 so we should be good to just re-run the build+upload